core/src/test/scala/unit/kafka/zk/ZkDataTest.scala

```diff
@@ -0,0 +1,25 @@
+package unit.kafka.zk
+
+import kafka.zk.ConsumerOffset
+import org.junit.{Assert, Test}
+
+class ZkDataTest {
+
+
+  @Test
```

Comment:
may be we can move this test to KafkaZkClientTest class.

Reply:
Moved to KafkaZkClientTest class